src/utils/formatNumbers.ts

```diff
@@ -458,31 +458,21 @@ function formatCurrencyAmount({
   })
 }
 
-function formatPriceImpact(priceImpact: Percent | undefined, locale: SupportedLocale = DEFAULT_LOCALE): string {
-  if (!priceImpact) return '-'
+function formatPercent(percent: Percent | undefined, locale: SupportedLocale = DEFAULT_LOCALE) {
+  if (!percent) return '-'
 
-  return `${Number(priceImpact.multiply(-1).toFixed(3)).toLocaleString(locale, {
-    minimumFractionDigits: 3,
+  return `${Number(percent.toFixed(3)).toLocaleString(locale, {
     maximumFractionDigits: 3,
     useGrouping: false,
   })}%`
 }
 
-function formatSlippage(slippage: Percent | undefined, locale: SupportedLocale = DEFAULT_LOCALE) {
-  if (!slippage) return '-'
-
-  return `${Number(slippage.toFixed(3)).toLocaleString(locale, {
-    maximumFractionDigits: 3,
-    useGrouping: false,
-  })}%`
-}
-
-function formatPercent(percent: Nullish<number>, locale: SupportedLocale = DEFAULT_LOCALE) {
-  if (percent === null || percent === undefined || percent === Infinity || isNaN(percent)) {
+function formatDelta(delta: Nullish<number>, locale: SupportedLocale = DEFAULT_LOCALE) {
```

Comment:
hmm the variable name `delta` is a bit vague, maybe we can call it something like `percentString`? either that or just leave a comment above describing what this one is for haha